docs/further.md

```diff
@@ -157,17 +157,14 @@ Some of these are [central snakemake command-line arguments determining its beha
   To reduce the burden of regular status checks,  this time automatically increases in 10 second steps with every status check whenever no finished jobs are found.
   But this will never go beyond a wait time of 180 seconds (3 minutes), to avoid long wait times once jobs do finish.
   Also, as soon as finished jobs are found, this gets reset to 40 seconds.
-  (TODO: respect the `--seconds-between-status-checks` option in the plugin and use it as the minimum wait time.)
```

Comment:
Should we add a new issue to track this TODO? Or have you addressed this, already?

Reply:
Ah wait, just saw your earlier comment. Will have another look, first.

Reply:
OK, this is now in the discussion below, so I'll resolve here.